packages/components/src/stories/SquiggleChart/Table.stories.tsx

```diff
@@ -0,0 +1,181 @@
+import type { Meta, StoryObj } from "@storybook/react";
+
+import { SquiggleChart } from "../../components/SquiggleChart.js";
+
+/**
+ * `Plot.scatter` objects in Squiggle are rendered with this chart.
+ */
+const meta = {
+  component: SquiggleChart,
+} satisfies Meta<typeof SquiggleChart>;
+export default meta;
+type Story = StoryObj<typeof meta>;
+
+export const Basic: Story = {
+  args: {
+    code: `
+    Table.make(
+      {
+        elements: [1, 4, 5],
+        columns: [
+          { fn: {|e|e}, name: "Number" },
+          { fn: {|e| normal(e^2, e^3)}, name: "Dist" },
+          { fn: {|e|[e, e, e, e]}, name: "Array" },
+          { fn: {|e|{first: e, second: e+1, third: e+2, fourth: e+3}}, name: "Record" },
+        ],
```

Comment:
This signature seems reasonable to me. Though `elements` is not very intuitive; I'm not sure if `rows` would be better or confusing in a different way.

Reply:
Changed to "data"